google/cloud/storage/blob.py

```diff
@@ -943,7 +937,25 @@ def _do_download(
             downloads where chunk_size is set) an INFO-level log will be
             emitted. Supported values are "md5", "crc32c" and None. The default
             is "md5".
+
+        :type retry: google.api_core.retry.Retry
+        :param retry: (Optional) How to retry the RPC. A None value will disable
+            retries. A google.api_core.retry.Retry value will enable retries,
```

Comment:
Does this change the default behavior to be no retries? I would expect the default value for this retry strategy to be some reasonable standard policy rather than None. Or is this okay here because the method is not one that end users will call directly?

Reply:
That last bit is correct; private methods such as this one don't have a retry default, but public methods do and they pass them through.